changelogs/fragments/1880-fix_cobbler_system_ssl.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  -  cobbler_sync, cobbler_system - fix SSL/TLS certificate check when ``validate_certs`` set to ``false`` (https://github.com/ansible-collections/community.general/pull/1880).
```

To make the linter happy:
```suggestion
  - cobbler_sync, cobbler_system - fix SSL/TLS certificate check when ``validate_certs`` set to ``false`` (https://github.com/ansible-collections/community.general/pull/1880).
```

---

Updated